File: public/docs/_examples/pipes/ts/app/power-booster.component.ts
// #docregion
import {Component} from 'angular2/core';
import {ExponentialStrengthPipe} from './exponential-strength.pipe';

@Component({
  selector: 'power-booster',
  template: `
    <h2>Power Booster</h2>
    <p>
      Super power boost: {{2 | exponentialStrength: 10}}
    </p>
  `,
  pipes: [ExponentialStrengthPipe]
})
export class PowerBooster { }

File: public/docs/_examples/pipes/dart/lib/power_booster_component.dart
// #docregion
import 'package:angular2/angular2.dart';
import 'exponential_strength_pipe.dart';

@Component(
    selector: 'power-booster',
    template: '''
      <h2>Power Booster</h2>
      <p>Super power boost: {{2 | exponentialStrength: 10}}</p>
    ''',
    pipes: const [ExponentialStrengthPipe])
class PowerBoosterComponent {}
